views/js/runner/provider/qti.js

```diff
@@ -491,15 +501,8 @@ define([
                     //set the plugin config from the test data
                     self.dataUpdater.updatePluginsConfig(self.getPlugins());
 
-                    //check if we need to trigger a storeChange
-                    if(!_.isEmpty(storeId) && !_.isEmpty(results.lastStoreId) && results.lastStoreId !== storeId){
-
-                        /**
-                         * We have changed the local storage engine (could be a browser change)
-                         * @event runner/provider/qti#storechange
-                         */
-                        self.trigger('storechange');
```

Comment:
Is it safe to remove this event? Or could this be considered a breaking change?

Reply:
I think this is ok since the plugins that listened to this event have also been updated accordingly